<launch>
    <arg name="output" default="screen"/>
    <arg name="min_z" default="0.5"/>
    <arg name="max_z" default="10.0"/>
    <arg name="source"/>
    <arg name="topic"/>

    <node pkg="pcl_assembler" type="pcl_filter_node" name="pcl_filter"
        ns="$(arg source)" output="$(arg output)">
        <remap from="cloud_in" to="$(arg topic)"/>
        <param name="min_z" type="double" value="$(arg min_z)"/>
        <param name="max_z" type="double" value="$(arg max_z)"/>
    </node>
</launch>
